### ISSUE TITLE
Documentation comments appear in reversed order.

### ISSUE THREAD

**Issue by regexident** [colin-kiegel/rust-derive-builder#27]
This …

```rust
struct Lorem {
    /// This is a doc comment for a field
    ///
    /// This is a second comment line
    field_with_doc_comment: String,
    ...
}
```

… turns into this …

```rust
struct Lorem {
    #[doc = r" This is a second comment line"]
    #[doc = r""]
    #[doc = r" This is a doc comment for a field"]
    field_with_doc_comment: String,
    ...
}
```

… when building with `--pretty=expanded`.

They then appear in the same reversed order within the HTML documents (via `$ cargo doc`).

**Comment by colin-kiegel** [comment]
Oops, we indeed missed that ^^

Thanks for the report.

As a workaround you could restrict your documentation to palindromes in the meantime.. just kidding.

**Comment by colin-kiegel** [comment]
fixed in v0.2.1 :-)
